Catappult/appcoins-wallet-android
# Creates wallet address if not exists before creating ticket

**What does this PR do?**

Prevents creating ticket without a wallet address. Uses existing
activity_iab_wallet_creation to show animation.
Uses existing method findWalletOrCreate for this.
Duplicate existing KeyboardUtils to hide keyboard when the user clicks
on the "find room" button.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

- Install 2048 and AppCoins Wallet
- Make sure that none of them has any user data (you can go to apps and clear data)
- Open 2048, insert a random username and click on "Find room". Wallet creation animation should appear.
- Payment methods dialog should appear shortly

**What are the relevant tickets?**

N/A

**Questions:**

N/A



**Code Review Checklist**
